### PR TITLE
Add Answer Properties component.

### PR DESCRIPTION
**Pull request by samiwel**
### What is the context of this PR?
image
This PR introduces the Answer Properties component.

### How to review 
All tests and checks should pass.
Should be able to use the answer properties component from within storybook.

### CODE REVIEW

**Comment by samiwel** [comment]
[Create AnswerProperties component (1)](https://trello.com/c/rXRXhfJ9/107-create-answerproperties-component-1)

File: src/components/AnswerProperties/story.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  max-width: 20em;
+`;
+
+const question = {
+  type: "General"
+};
+
+storiesOf("Answer Properties", module).add("Default", () =>
+  <Background>
+    <AnswerProperties question={question} />
```
Comment (hamishtaplin) [inline src/components/AnswerProperties/story.js]: Should the story implement and log the event handling?

File: src/components/AnswerProperties/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import styled from "styled-components";
+import { colors } from "constants/theme";
+
+const StyledCheckboxInput = styled(Input)`
+  border: 2px solid ${colors.borders};
+  height: 1.4em;
+  width: 1.4em;
+  margin-right: 0;
+`;
+const FlexField = styled.div`
```
[the remark this answers is not in the document]
Comment (samiwel) [reply src/components/AnswerProperties/index.js]: Realised that Field was not passing down `className`. Fixed now.

File: src/components/AnswerProperties/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      </FlexField>
+      <Field id="answer.validation.message">
+        <Label>Custom validation message</Label>
+        <TextArea placeholder="Optional" {...props} />
+      </Field>
+    </Form>
+  );
+};
+
+AnswerProperties.propTypes = {
```
[the remark this answers is not in the document]
Comment (samiwel) [reply src/components/AnswerProperties/index.js]: Yes, I have made a few required.

File: src/components/AnswerProperties/story.js (hunk 1 above)
Comment (samiwel) [reply src/components/AnswerProperties/story.js]: This is being done elsewhere, If it doesn't handle the onChange, then you'd never see the value being updated.

**Pull request merged**